apps/web/app/components/blocks/structure/MultiGrid/MultiGrid.vue

```diff
@@ -146,4 +143,63 @@ const columns = computed<Block[][]>(() => {
   });
   return blocks.value;
 });
+
+const containsItemGrid = computed(() => {
+  return hasItemGridInColumns(columns.value);
+});
+
+const getBlockActions = (block: Block) => {
+  if (block.name === 'ItemGrid') {
+    return {
+      isEditable: true,
+      isMovable: false,
+      isDeletable: false,
+      classes: ['bg-purple-400', 'hover:bg-purple-500', 'transition'],
+      buttonClasses: ['border-2', 'border-purple-600'],
+      hoverBackground: ['hover:bg-purple-500'],
+      position: `${currentTop.value}px`,
+    };
+  }
+
+  return {
+    isEditable: true,
+    isMovable: false,
+    isDeletable: false,
+    classes: ['bg-purple-400', 'hover:bg-purple-500', 'transition'],
+    buttonClasses: ['border-2', 'border-purple-600'],
+    hoverBackground: ['hover:bg-purple-500'],
+    position: '',
+  };
+};
+watch(
```

Comment:
Should we include a check for edit/live mode here? The way I understand it, the watcher will get constantly triggered in production as well right now, even though it doesn't have any practical effect.